kodilla-patterns2/src/main/java/com/kodillapatterns2/observer/homework/Mentor.java

```diff
@@ -0,0 +1,24 @@
+package com.kodillapatterns2.observer.homework;
+
+public class Mentor implements HomeworkObserver {
+    private String name;
+    private int updateCount;
+
+    public Mentor(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getUpdateCount() {
+        return updateCount;
+    }
+
+    @Override
+    public void update(HomeworkQueue homeworkQueue) {
+        System.out.println(name + ": New homework added to " + homeworkQueue.getStudent().getName() + "'s queue: " + homeworkQueue.getLastAddedHomework().getTaskName());
```

Comment:
zmienna lastAddedHomework I uzycie getLastAddedHomework jest tutaj ryzykowane. Jesli nie ma ani jednego homework dodanego, to poleci tutaj Null Pointer Exception

Reply:
Ok, usunąłem z kodu